Makefile

```diff
@@ -141,7 +141,7 @@ install:
 	@sh -c '. ./packaging/functions.sh; install_data $(CWD) $(OPENRA_INSTALL_DIR) cnc d2k ra'
 
 install-linux-shortcuts:
-	@sh -c '. ./packaging/functions.sh; install_linux_shortcuts $(CWD) $(OPENRA_INSTALL_DIR) $(BIN_INSTALL_DIR) $(DATA_INSTALL_DIR) $(VERSION) cnc d2k ra'
+	@sh -c '. ./packaging/functions.sh; install_linux_shortcuts $(CWD) $(DESTDIR) $(gameinstalldir) $(bindir) $(datadir) $(VERSION) cnc d2k ra'
```

Comment:
DESTDIR isn't defined. Shouldn't this at least default to an empty string?

Reply:
Idk but fyi https://www.gnu.org/prep/standards/html_node/DESTDIR.html#DESTDIR  says `You should not set the value of DESTDIR in your Makefile at all`.

Reply:
My concern is that if this is empty it may throw out the order of the arguments behind it. I'm not sure how bash handles this case.  Wrapping each of these arguments in "" is a good idea in any case and should also solve this.

Reply:
Added the additional safety.